requirements/base.in

```diff
@@ -9,6 +9,7 @@ djangorestframework-jsonapi==5.0.0
 edx-drf-extensions
 edx-i18n-tools
 edx-opaque-keys
+eox-audit-model
```

Comment:
So you prefer eox-audit now is mandatory to the plugin?

Reply:
I didn't consider that dependency was optional, I thought the current implementation was designed in that way due to the tests, probably in the future this should be mandatory since this is a plugin designed for NELC and doesn't make sense to make that optional for other installations when we only have one however that should be applied in another pr